web/source/osk/activeLayout.ts

```diff
@@ -389,9 +394,9 @@ namespace com.keyman.osk {
     }
 
     /**
-     * 
+     *
      * @param layout
-     * @param formFactor 
+     * @param formFactor
```

Comment:
Is this a "public API" function? If so, add some genuine documentation! If it's a function that is only used, say in this file, then just delete the doc comment.

Reply:
I didn't add this documentation -- you are looking at whitespace changes! -- there is a general inconsistency in commenting. I am a big believer in adding comments for internal functions as well though -- very useful to people working on the code.